### PR TITLE
Checkout: update creditcards to gain ELO support, add test

### PR DESCRIPTION
**Pull request by yoavf**
We'd like to be able to detect the ELO credit card type.

### CODE REVIEW

**Comment by yoavf** [comment]
:( v2.0.0 of `creditcards-types` is now returning an array and relies on array.find() which we don't have a polyfill for.

**Comment by yoavf** [comment]
Added a polyfill (similar to how it was done for draft-js)

File: client/components/payment-logo/style.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	vertical-align: middle;
 	width: 35px;
 
-	$cc-payment-types: 'amex', 'diners', 'discover', 'jcb', 'mastercard', 'unionpay', 'visa';
+	$cc-payment-types: 'amex', 'diners', 'discover', 'elo', 'jcb', 'mastercard', 'unionpay', 'visa';
```
Comment (akirk) [inline client/components/payment-logo/style.scss]: What about renaming this to `cc-brands` as well?

**Comment by yoavf** [comment]
This is delayed, but I need to go back to this. We've been able to charge ELO cards already, so I need to confirm the BIN numbers, and if the update to the module is necessary.

**Comment by alisterscott** [comment]
This branch needs a rebase. Do we still need this PR?

**Comment by yoavf** [comment]
Closing for now.

**Pull request closed without merging**